src/server/unit_sao.cpp

```diff
@@ -75,11 +75,13 @@ void UnitSAO::setAnimationSpeed(float frame_speed)
 	m_animation_speed_sent = false;
 }
 
-void UnitSAO::setBonePosition(const std::string &bone, v3f position, v3f rotation)
+void UnitSAO::setBonePosition(
+		const std::string &bone, v3f position, v3f rotation, bool stop_animations)
 {
 	// store these so they can be updated to clients
 	m_bone_position[bone] = core::vector2d<v3f>(position, rotation);
 	m_bone_position_sent = false;
+	m_stop_animations = stop_animations;
```

Comment:
Since this is an object-wide property it should either be moved to the object properties or properly made to work per-bone.